src/runtime_src/core/edge/drm/zocl/zocl_sysfs.c

```diff
@@ -133,6 +134,60 @@ static ssize_t zocl_get_memstat(struct device *dev, char *buf, bool raw)
 	return size;
 }
 
+static ssize_t graph_status_show(struct device *dev, struct device_attribute *attr, char *buf)
+{
+	struct drm_zocl_dev *zdev;
+	size_t size;
+	struct aie_info *aie;
+	struct aie_info_cmd *acmd;
+	struct aie_info_packet *aiec_packet;
+	ssize_t nread = 0;
+	zdev = dev_get_drvdata(dev);
+	if (!zdev)
+		return 0;
+
+	aie = zdev->aie_information;
+	acmd = kmalloc(sizeof(struct aie_info_cmd), GFP_KERNEL);
```

Comment:
Please add some one line comments regarding what is happening at various points in this function.

Reply:
added

Reply:
A few more comments would be useful.